elasticms-cli/src/Command/FileReader/FileReaderImportCommand.php

```diff
@@ -104,6 +130,20 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         }
         $progressBar->finish();
 
+        if ($this->dryRun && \count($ouuids) > 0) {
+            $this->io->write(PHP_EOL.PHP_EOL);
+            $this->io->warning(\sprintf('%d documents are missing in the source file and will be deleted without the %s option', \count($ouuids), self::OPTION_DRY_RUN));
+        } elseif (\count($ouuids) > 0) {
+            $this->io->write(PHP_EOL.PHP_EOL);
```

Comment:
Same? so the $this->io->newLine(2); can be before the if check

Reply:
I don't need those new lines outside those ifs. Refactored (tks I learned something)